sandbox-templates/mdx-evaluate-create-react-app/src/App.js

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from "react";
+import * as runtime from "react/jsx-runtime";
+import { evaluate } from "@mdx-js/mdx";
+
+const mdxContent = `
+# heading
+
+* list
+* item
+
+\`\`\`js
+function () {}
+\`\`\`
+`;
+
+function useMDX(content) {
```

Comment:
The downside of this, is that due to its name, it makes it "recommended". But it doesn't allow a cool thing about mdx 2: accessing other exports. Maybe it's fine. Maybe it hides that coolness?

Reply:
How about 01d7be6?